### PR TITLE
remove script tag

### PR DESCRIPTION
**Pull request by torhoehn**
### Description (*)

Removed script tag, but tooltip still opens correctly.

### Fixed Issues (if relevant)

1. Fixes magento/magento2#25050 : Product AdditionalOptions "Read more" dots being escaping on invoice/shipment overview pages

### Manual testing scenarios (*)

1. Create a product that includes an additionalOption value that is over 55 characters
2. Place order for product
3. View invoice for order in customer dashboard

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by kalpmehta** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @kalpmehta. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @kalpmehta, here is your [new Magento instance](https://pr-25418.instances.magento-community.engineering).
Admin access: [https://pr-25418.instances.magento-community.engineering/admin](https://pr-25418.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Review by kalpmehta — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @kalpmehta, thank you for the review.
[ENGCOM-6223](https://jira.corp.magento.com/browse/ENGCOM-6223) has been created to process this Pull Request
:eight_spoked_asterisk: @kalpmehta, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Review by dmytro-ch — CHANGES_REQUESTED** [review]
Hi @torhoehn,
could you please make sure the tooltip is also initialized for order items renderer as well for shipment, invoice and credit memo.
So we would get the same behavior for all tabs.
Screenshot 2019-11-01 at 09 04 29
Screenshot 2019-11-01 at 09 04 38

E.g.
Shipment Items: `Magento/Sales/view/frontend/templates/order/shipment/items/renderer/default.phtml`
Order Items: `Magento/Sales/view/frontend/templates/order/items/renderer/default.phtml`

**Comment by torhoehn** [comment]
@dmytro-ch You're absolutely right and I added it.

**Comment by dmytro-ch** [comment]
@torhoehn, great, thank you!

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6223](https://jira.corp.magento.com/browse/ENGCOM-6223) has been created to process this Pull Request

**Comment by engcom-Bravo** [comment]
Hello @torhoehn. Thank You for contribution. The PR fixes the issue - that's Ok. But I'd like to return for a wile to Your previous PR (or one of the previous PRs) #25393. It has already been merged and on the latest develop it works just as expected. But now as I tested ENGCOM-6223 branch with Your current PR the old issue with the tooltip returns in it. And that issue encounters only on the Order Shipments tab:
tooltip
Let me summarize all said about. So,
1. We are talking about old issue: "Product Additional Options Tooltip not fully visible #25392"
2. Your previous PR "fix additional product options tooltip visibility #25393" has fixed it and now in the develop version the issue is not reproduced any more.
3. For the current PR proccesing a [branch](https://github.com/magento/magento2/pull/25418#issuecomment-550222442) of ENGCOM-6223 was created.
4. On ENGCOM-6223 branch the old issue of #25392 appears again but only in relation to Order Shipments tab.
Shan't merging of the current PR into develop break the correct behavior we now have there?

**Comment by torhoehn** [comment]
@engcom-Bravo This expected behaviour, because until now the fix #25393 is not inside of this branch. If you want I could merge 2.3-develop into this feature, so you we can see if everything works as it should.

**Comment by engcom-Bravo** [comment]
Ok @torhoehn, please take care from Your side as much as You can on that all the fixes will persist in the develop. Thank You.  :heavy_check_mark: QA Passed.

**Comment by torhoehn** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @torhoehn. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @torhoehn, here is your [new Magento instance](https://pr-25418.instances.magento-community.engineering).
Admin access: [https://pr-25418.instances.magento-community.engineering/admin](https://pr-25418.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by torhoehn** [comment]
@engcom-Bravo Everything works as expected now!

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6223](https://jira.corp.magento.com/browse/ENGCOM-6223) has been created to process this Pull Request

File: app/code/Magento/Sales/view/frontend/templates/order/creditmemo/items/renderer/default.phtml [hunk 1]
```diff
@@ -17,7 +17,7 @@
                 <?php if (!$block->getPrintStatus()) : ?>
                     <?php $_formatedOptionValue = $block->getFormatedOptionValue($_option) ?>
                     <dd<?= (isset($_formatedOptionValue['full_view']) ? ' class="tooltip wrapper"' : '') ?>>
-                        <?= $block->escapeHtml($_formatedOptionValue['value']) ?>
+                        <?= $block->escapeHtml($_formatedOptionValue['value'], ['a', 'img']) ?>
```
Comment (oleksii-lisovyi) [inline app/code/Magento/Sales/view/frontend/templates/order/creditmemo/items/renderer/default.phtml:20]: `img` tag is not allowed in the html Escaper class:
https://github.com/magento/magento2/blob/2.4-develop/lib/internal/Magento/Framework/Escaper.php#L40

This generates an error `The following tag(s) are not allowed: img` when go to the Order page in My Account
The line in the Escaper:
https://github.com/magento/magento2/blob/2.4-develop/lib/internal/Magento/Framework/Escaper.php#L457

Probably removing the `img` from allowed tags passed as the second parameter should be enough.

Comment (themystery404) [reply app/code/Magento/Sales/view/frontend/templates/order/creditmemo/items/renderer/default.phtml:20]: I created pull request, that fixed critical error 
https://github.com/magento/magento2/pull/29863

**Pull request merged**